src/test/java/uk/gov/companieshouse/uri/web/transformer/CompanyDetailsTransformerImplTest.java

```diff
@@ -166,6 +166,17 @@ void profileApiToDetailsNullAccountingReferenceDateApi() {
         assertEquals(null, companyDetails.getAccounts().getAccountRefMonth());
     }
     
+    @Test
+    void profileApiToDetailsMissingLastAccounts() {
+        CompanyProfileApi companyProfileApi = populatedCompanyProfileApi();
+        companyProfileApi.getAccounts().setLastAccounts(null);;
+
+        CompanyDetails companyDetails = testCompanyDetailsTransformer.profileApiToDetails(companyProfileApi);
+        
+        assertEquals(null, companyDetails.getAccounts().getLastMadeUpDate());
```

Comment:
I think you may be able to use assertNul() here instead of assertEquals().

Reply:
Good point, thanks. I changed all relevant assertEquals to assertNull in that test class.